cmd/image-builder/main.go

```diff
@@ -822,10 +808,35 @@ func main() {
 		}
 		os.Exit(0)
 	}
+
 	err = buildLocally(o)
 	if err != nil {
 		fmt.Println(err)
 		os.Exit(1)
 	}
 	fmt.Println("Job's done.")
 }
+
+func parseTagsFromEnv(o options) ([]tags.Tag, error) {
```

Comment:
Same comment about `FromEnv` as for `LoadGitStateConfigFromEnv` function. Please consider removing `FromEnv` suffix.